Automattic/jetpack
# PHPCS: SEO Tools

Fixes #21969

#### Changes proposed in this Pull Request:

PHPCS for SEO Tools files.

#### Jetpack product discussion

#21969

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proofread changes.
* Pull down changes, make sure PHP errors/warnings are logged, and test that SEO settings work as expected on `/wp-admin/admin.php?page=jetpack#/traffic`, and then try again from Calypso which shouldn't be any different `tools > marketing > traffic`.
